database/migrations/2019_07_17_074500_create_stripe_accounts_and_events.php

```diff
@@ -61,7 +61,7 @@ public function up()
             $table->timestamps();
             $table->string('account_id', 255)->nullable()->collate(LaravelStripe::ID_DATABASE_COLLATION);
             $table->date('api_version');
-            $table->timestamp('created');
+            $table->timestamp('created')->nullable();
```

Comment:
Question: when would this be nullable? The Stripe API docs doesn't say it can be `null`?